recipes/voyager/recipe.yaml

```
@@ -0,0 +1,45 @@
+context:
+  version: 2.1.0
+
+package:
+  name: voyager
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/spotify/voyager/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: 5f8d687f310b67e9661bb6226b68eea4c8d0c5d48f5866be11c8a37884ce6ed1
+
+build:
+  number: 0
+  script:
+    - cd python
+    - ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - ${{ compiler('cxx') }}
+    - pip
+    - setuptools
+    - numpy
+    - pybind11
```

I know `${{ compiler('cxx') }}` needs to be in the build section. I *think* the correct syntax to use is as below:
```suggestion
requirements:
  build:
    - if: build_platform != target_platform
      then:
        - python
        - cross-python_${{ target_platform }}
        - numpy
        - pybind11
        - setuptools
    - ${{ compiler('cxx') }}  
  host:
    - python
    - pip
    - numpy
    - pybind11
    - setuptools
```